core/src/main/scala/algebra/ring/Ring.scala

```diff
@@ -74,4 +74,39 @@ trait RingFunctions[R[T] <: Ring[T]] extends AdditiveGroupFunctions[R] with Mult
 
 object Ring extends RingFunctions[Ring] {
   @inline final def apply[A](implicit ev: Ring[A]): Ring[A] = ev
+
+  /** Returns the given Double, understood as a rational number, in the provided
+    * (division) ring.
+    * 
+    * This is implemented in terms of basic ops. However, this is
+    * probably significantly less efficient than can be done with a specific
+    * type. So, it is recommended to specialize this general method.
+    */
+  def fromDouble[A](a: Double)(implicit ringA: Ring[A], mgA: MultiplicativeGroup[A]): A =
```

Comment:
I just noticed this was in algebra, not spire! So, yeah, I guess Ring isn't too bad here. But I still think we should rename this `defaultFromDouble`.